internal/metrics/client.go

```diff
@@ -95,7 +114,7 @@ func (c *client) Measure(b ginkgo.Benchmarker, f queryFunc, name string, job str
 
 func (c *client) ContainerUserCPU(caJob string, duration model.Duration) (float64, error) {
 	query := fmt.Sprintf(
-		`sum(rate(container_cpu_user_seconds_total{job="%s"}[%s]) * 1000)`, // in m (i.e. 1000 = 1 vCore)
```

Comment:
This makes this OCP-only specific. Can we keep please the independence from OCP.

Reply:
The problem is that `job` doesn't work for the OCP case. The only way to tell the difference is with the `pod` key. So, in this case the `pod` key must be in there for this to work.

Reply:
If it doesn't work for OCP, but works for our default-prometheus, we need to make this configurable. We certainly want to keep the ootb prometheus experience to continue to work as it is.